libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -181,6 +183,7 @@ class MapboxNavigation(
     private val tripSession: TripSession
     private val navigationSession: NavigationSession
     private val logger = LoggerProvider.logger
+    private val connectivityHandler: ConnectivityHandler = ConnectivityHandler()
```

Comment:
As far as I can tell from only looking at the code, this handler instance is never notified, right?

The `ReachabilityService` creates another instance in place, so this one is never used.